source/_docs/scripts/conditions.markdown

```diff
@@ -32,7 +32,7 @@ The following configuration works the same as the one listed above:
 
 ```yaml
 alias: "Paulus home AND temperature below 20"
-conditions:
+condition:
```

Comment:
This is not correct.
This example is a block that should go under a `condition:` key just like the example over it.

This would be the full example:
```yaml
condition:
  alias: "Paulus home AND temperature below 20"
  conditions:
    - condition: state
      entity_id: "device_tracker.paulus"
      state: "home"
    - condition: numeric_state
      entity_id: "sensor.temperature"
      below: 20
```

Reply:
@ludeeus Can you double check? Because: 

- I've just tested with a similar case your proposal and reloading automation raised an error.
- Creating an automation in the UI with two conditions will generate a YAML similar to my proposal and without error when reloading automation.

Reply:
Yeah this is all sorts of wrong

```
condition:
  - condition: and
    alias: "Paulus home AND temperature below 20"
    conditions:
      - condition: state
        entity_id: "device_tracker.paulus"
        state: "home"
      - condition: numeric_state
        entity_id: "sensor.temperature"
        below: 20
```
or 
```
condition:
    alias: "Paulus home AND temperature below 20"
    - condition: state
      entity_id: "device_tracker.paulus"
      state: "home"
    - condition: numeric_state
      entity_id: "sensor.temperature"
      below: 20
```

These are both valid

Reply:
Your second example still raise an error during configuration validation. It doesn't like mixing the mapping and the list. The `alias` key should be elsewhere.

It seems my initial proposal is the good one. Or we need to fix something in configuration validation mechanism.

Reply:
Right, it worked in the UI.
Use the second and drop the alias from this example

Reply:
Done